test/e2e/framework/size.go

```diff
@@ -128,3 +139,16 @@ func WaitForGroupSize(group string, size int32) error {
 	}
 	return fmt.Errorf("timeout waiting %v for node instance group size to be %d", timeout, size)
 }
+
+func getGCEZoneForGroup(group string) (string, error) {
+	zone := TestContext.CloudConfig.Zone
+	if TestContext.CloudConfig.MultiZone {
+		output, err := exec.Command("gcloud", "compute", "instance-groups", "managed", "list",
+			"--project="+TestContext.CloudConfig.ProjectID, "--format=value(zone)", "--filter=name="+group).CombinedOutput()
```

Comment:
Have you checked that "--format=value(zone)" works? When I tried doing it on my cmdline, it complains with `bash: syntax error near unexpected token `('`. I had to put `value(zone)` inside single quotes to make it work.

Reply:
Yep, tested this and it actually works (exec.Conmand takes care of this?). See similar use in test-infra code: https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L416